homeassistant/components/led_ble/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "led_ble",
+  "name": "LED BLE",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ble_ble",
+  "requirements": ["led-ble==0.5.4"],
+  "dependencies": ["bluetooth"],
+  "codeowners": ["@bdraco"],
+  "bluetooth": [
+    { "local_name": "LEDnet*" },
+    { "local_name": "BLE-LED*" },
+    { "local_name": "LEDBLE*" },
+    { "local_name": "Triones*" },
+    { "local_name": "LEDBlue*" }
+  ],
+  "iot_class": "local_polling"
```

Comment:
I plan on adding these as supported brands in the future once I have a few more of the devices in hand. ... Aliexpress shipping is slow